draft-ietf-tls-esni.md

```diff
@@ -791,6 +793,13 @@ where HKDF-Expand-Label and HKDF-Extract are as defined in {{RFC8446}}. The
 value of ServerHello.random[0:24] is generated as usual by invoking a secure
 random number generator (see {{RFC8446}}, Section 4.1.2).
 
+\[\[TODO: If the ClientHello offers TLS 1.2 or below, it's possible the backend
```

Comment:
Which ClientHello?

Reply:
Presumably the inner, given below.

Reply:
Yeah, the backend server doesn't see the outer one.